### PR TITLE
made solution

### PR DESCRIPTION
**Pull request by Sashk52**

### CODE REVIEW

File: src/main/java/taxi/controller/AddDriverController.java [hunk 1]
```diff
@@ -27,7 +27,12 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
...
         String password = req.getParameter("password");
-        Driver driver = driverService.create(new Driver(name,licenceNumber,login,password));
-        resp.sendRedirect(req.getContextPath() + "/drivers/all");
+        String repeatPassword = req.getParameter("repeat_password");
+        if (password.equals(repeatPassword)) {
+            driverService.create(new Driver(name, licenceNumber, login, password));
+            resp.sendRedirect(req.getContextPath() + "/drivers/login");
+        }
+        req.setAttribute("error Message", "Passwords are not identical!");
+        req.getRequestDispatcher("/WEB-INF/views/drivers/login");
```
Comment (mfedenko09) [inline src/main/java/taxi/controller/AddDriverController.java:36]:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/drivers/login");
```
This won't work without .forward() I think

**Review by mfedenko09 — APPROVED** [review]

File: src/main/webapp/WEB-INF/views/cars/driver/all.jsp [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Sashk
+  Date: 26.01.2021
+  Time: 0:33
+  To change this template use File | Settings | File Templates.
+--%>
```
Comment (RomanDubovskyi) [inline src/main/webapp/WEB-INF/views/cars/driver/all.jsp]: remove autogenerated comment

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]
you don't have filter in PR, this task is not fully completed

File: src/main/webapp/WEB-INF/views/cars/driver/all.jsp (hunk 2 above)
Comment (Sashk52) [reply src/main/webapp/WEB-INF/views/cars/driver/all.jsp]: done

**Pull request closed without merging**